docs/index.rst

```diff
@@ -3,7 +3,7 @@ Solidity
 
 Solidity is a high-level language whose syntax is similar to that of JavaScript
 and it is designed to compile to code for the Ethereum Virtual Machine.
-As you will see, it is quite easy to create contracts for voting,
+As you will see, it is possible to create contracts for voting,
```

Comment:
Whether it's easy or not, how does it serve us to state that this is "quite easy" to create these contracts? Easy compared to what? 

I've found that there is a pervasive sentiment of "easy" and "simple" in Ethereum. This leads to two problems:
* When newcomers inevitably find that it's not easy, they question whether they're cut out for Ethereum.
* More experienced smart contract developers get over-confident, instead of taking a more critical look at their code.